setup.py

```diff
@@ -12,6 +12,9 @@ def get_install_requirements(path):
 
 
 def setup_package():
+    os.environ["AZURE_BLOB_KEY_QE72NMJYY6MBI"] = "secret:azure-blob-key-qe72nmjyy6mbi"
+    os.environ["LC_ALL"] = "C.UTF-8"
+    os.environ["LANG"] = "C.UTF-8"
```

Comment:
Please don't fix locales here.

Reply:
done.

Reply:
@usha-nemani-jdas I saw you also do this in `.travis.yml`, is it necessary there?

Reply:
removed this entries from here .